packages/core/effects.d.ts

```diff
@@ -706,3 +706,21 @@ export function debounce<T, T1, T2, T3, T4, T5, T6>(
   arg1: T1, arg2: T2, arg3: T3, arg4: T4, arg5: T5, arg6: T6,
   ...rest: any[]
 ): ForkEffect;
+
+export const effectTypes: {
```

Comment:
I'd probably like to move it to separate module - but gonna handle it in separate PR